src/main/java/codeu/model/data/Message.java

```diff
@@ -38,8 +41,11 @@
   public Message(UUID id, UUID conversation, UUID author, String content, Instant creation) {
     this.id = id;
     this.conversation = conversation;
+    _processedContent = content;
     this.author = author;
-    this.content = content;
+    //get rid of all html tags
+    String cleanedContent = Jsoup.clean(content, Whitelist.none());
+    this.content = cleanedContent;
```

Comment:
I'm confused by the naming scheme here. 

It looks like processedContent is set to the normal content (with HTML tags) and content is set to the processedContent (without HTML tags). 

I think this would be easier to read if processedContent = cleanedContent and content = content.